localPa11y.json

```diff
@@ -3,12 +3,21 @@
         "chromeLaunchConfig": {
             "ignoreHTTPSErrors": true,
             "args": ["--disable-dev-shm-usage", "--no-sandbox"]
-    }
-},
+        },
+        "useIncognitoBrowserContext": false
+    },
     "urls": [
-        "http://localhost:3000/",
+        {
+            "url": "http://localhost:3000/login",
+            "actions": [
```

Comment:
Do you actually need this fix anymore?

Reply:
the file was updated from mohammed, so I just added it from him